cmd/vc-rest/startcmd/start_test.go

```diff
@@ -160,6 +162,9 @@ func TestStartCmdValidArgsEnvVar(t *testing.T) {
 	err = os.Setenv(blocDomainEnvKey, "domain")
 	require.Nil(t, err)
 
+	err = os.Setenv(universalResolverURLEnvKey, "localhost:8083")
```

Comment:
it's a good idea to defer the removal of these test environment variables

Reply:
Done!